lib/HotKeys.js

```diff
@@ -118,7 +118,17 @@ class HotKeys extends React.Component {
   }
 
   componentDidMount() {
-    this.attachment = this.props.attach || ReactDOM.findDOMNode(this);
+    const { attach } = this.props;
+
+
+
+    // handle refs...
+    if (attach) {
+      this.attachment = attach?.current ? attach.current : attach;
+    } else {
+      // this will still error if no attach prop is provided...
+      this.attachment = ReactDOM.findDOMNode(this);
+    }
```

Comment:
Should this be `else { throw Error(); }`? `findDOMNode()` will be deprecated. You've done a nice job building in a fallback to continue supporting the previous API in order to avoid a breaking change, but I wonder if this just needs to be a breaking change.

Reply:
I don't want to force it that hard while the advent of React 19 is still unknown. For now it would enable the cleanup to happen while we still maintain the working state of Hotkeys. It will be a breaking change eventually... I feel like a number of things probably will once we do adopt v19 full on...

Reply:
Good thinking; this approach will ease the migration, allowing it to happen on a per-component basis rather than forcing all components to upgrade at once.

Related, sorta: I don't remember the whole history here, but it appears the only consumer of this package is `<HotKeys>` in stripes-components. Is there value in continuing to maintain this as a separate package or could/should we just fold it in since it is already just an internal implementation detail? Seems like the "migration" would literally just be a change in the import path.

Reply:
That's a good point... all things through this are prettymuch consumed through stripes-components' offerings. If it's still that way this late in the game, no reason why we wouldn't just close the book on this outlier.